extension/llm/test/test_files/basic.test

```diff
@@ -0,0 +1,73 @@
+
+-CASE SemanticDistanceCheck
+
+-STATEMENT load extension "${KUZU_ROOT_DIRECTORY}/extension/llm/build/libllm.kuzu_extension"
+---- ok
+
+-LOG Similar Text 1
+-STATEMENT WITH create_embedding("Who is the author of 1984?") AS t1, create_embedding("Who wrote the novel 1984?") AS t2 RETURN array_distance(t1, t2) AS distance;
+---- 1
+7.589955
+
+-LOG Similar Text 2
+-STATEMENT WITH create_embedding("What is the capital of France?") AS t1, create_embedding("Which city is the capital of France?") AS t2 RETURN array_distance(t1, t2) AS distance;
+---- 1
+4.689638
+
+-LOG Dissimilar Text 1
+-STATEMENT WITH create_embedding("How to train a dog") AS t1, create_embedding("What is the capital of France") AS t2 RETURN array_distance(t1, t2) AS distance;
+---- 1
+26.315777
+
+-LOG Dissimilar Text 2
+-STATEMENT WITH create_embedding("How do Unix file permissions work?") AS t1, create_embedding("History of the Roman Empire") AS t2 RETURN array_distance(t1, t2) AS distance;
```

Comment:
Would have liked to see this used in a `vector` test. But we can put that in a later PR.